docs/docs/mdx/markdown-syntax.md

```diff
@@ -57,13 +62,15 @@ Here's how those tags render in HTML:
 
 - can use `*`, `-`, or `+` for each list item
 
-```
+```markdown
+- Gatsby
+  - docs
+
 * Gatsby
-  * docs
+  - docs
+
 - Gatsby
   - docs
-+ Gatsby
-  + docs
 ```
```

Comment:
linting changed here the examples of `*` and `+` to `-`
:/

Reply:
reverted the changes. added `prettier-ignore`

Reply:
Huh, prettier does this (good that I still not use prettier)? Which rule causes this change?

Reply:
Can you squash (`git rebase -i ...` with `fixup`) the commits thens?

Reply:
squashed the many revert commits into one